ambry-replication/src/main/java/com.github.ambry.replication/ReplicaTokenPersistor.java

```diff
@@ -155,7 +154,11 @@ public void serializeTokens(List<ReplicaTokenInfo> tokenInfoList, OutputStream o
           writer.write(replicaTokenInfo.getReplicaPath().getBytes());
           // Write port
           writer.writeInt(replicaTokenInfo.getPort());
+          // Write total bytes read from local store
           writer.writeLong(replicaTokenInfo.getTotalBytesReadFromLocalStore());
+          // Write replica type
+          writer.writeShort((short) replicaTokenInfo.getReplicaInfo().getReplicaId().getReplicaType().ordinal());
```

Comment:
do these changes require a version update? Since the format without these changes is currently on existing servers?